frontend/src/components/Scopes.tsx

```diff
@@ -0,0 +1,181 @@
+import React from "react";
+import { Checkbox } from "src/components/ui/checkbox";
+import { Label } from "src/components/ui/label";
+import { cn } from "src/lib/utils";
+import {
+  NIP_47_MAKE_INVOICE_METHOD,
+  NIP_47_NOTIFICATIONS_PERMISSION,
+  NIP_47_PAY_INVOICE_METHOD,
+  SCOPE_GROUP_CUSTOM,
+  SCOPE_GROUP_ONLY_RECEIVE,
+  SCOPE_GROUP_SEND_RECEIVE,
+  Scope,
+  ScopeGroupType,
+  WalletCapabilities,
+  scopeDescriptions,
+  scopeGroupDescriptions,
+  scopeGroupIconMap,
+  scopeGroupTitle,
+} from "src/types";
+
+// TODO: this runs everytime, use useEffect
+const scopeGrouper = (scopes: Set<Scope>) => {
+  if (
+    scopes.size === 2 &&
```

Comment:
why would our default group only have these 2 scopes? I would expect the default group to have all scopes